sql/pgwire/server.go

```diff
@@ -36,6 +37,8 @@ const (
 	versionSSL = 80877103
 )
 
+const pgPort = 15432
```

Comment:
Seems a bit strange that we're duplicating the port here and in `server/context.go`. Can we move this logic into `server` near the usage of `context.PGAddr`? And why is this necessary for `context.PGAddr` but not `context.Addr`? Or maybe it is necessary for `context.Addr` but I'm just not seeing where it is done.